### ISSUE TITLE
[BUG] - 7za missing in the compiled macOS version

### ISSUE THREAD

**Issue by nezbite** [YourKalamity/lazy-dsi-file-downloader#11]
**Describe the bug**
When on macOS `7za` from the `Darwin` folder is used. This works when using the python files, but does not when running the executable for macOS since there is no Darwin folder next to that file.

**To Reproduce**
Open the macOS release version on macOS. duh

**Expected behavior**
The compiled version should either include the 7za in itself (usually the case in macOS executables) or the .zip file the macOS release version comes in should include the `Darwin/7za` folder (probably easier for now). I have tested copying the `Darwin` folder next to the compiled version and that works.

**Desktop (please complete the following information):**
 - OS: macOS Big Sur 11.0.1

I know developing on macOS [can be a pain](https://github.com/YourKalamity/lazy-dsi-file-downloader/blob/2a7bfd934ebdfd8eb32f1f6bae8661547c020bde/main.py#L658). When you need help with testing fixes I am glad to help you out.

**Comment by nezbite** [comment]
I have seen multiple issues in this repository regarding 7zip. As a suggestion, if no 7zip is found on your system you could download it in the downloading process.

**Comment by YourKalamity** [comment]
Strange, the Darwin folder should be included _inside_ the executable though I don't have much experience with macOS executables.  Downloading it is a good suggestion, I will try implementing it in the Qt release.

**Comment by agnesnutter** [comment]
Still seen in 3.1.4:

```
Exception in thread Thread-1 (start):
Traceback (most recent call last):
  File "threading.py", line 1009, in _bootstrap_inner
  File "threading.py", line 946, in run
  File "main.py", line 128, in start
FileNotFoundError: [Errno 2] No such file or directory: 'Darwin/7za'
```

**Comment by YourKalamity** [comment]
Should be fixed with a618d49288186ad942e9aaecf6d18eff68a3994a
